implementation/src/main/java/io/smallrye/faulttolerance/tracing/TracingInstaller.java

```diff
@@ -0,0 +1,25 @@
+package io.smallrye.faulttolerance.tracing;
+
+import org.jboss.logging.Logger;
+
+/**
+ * @author Pavol Loffay
+ */
+public class TracingInstaller {
+
+  private static final Logger LOGGER = Logger.getLogger(TracingInstaller.class);
+
+  private TracingInstaller() {}
+
+  /**
+   * Install installs tracing if OpenTracing libraries are on classpath.
+   */
+  public static void install() {
+    try {
+      Class.forName("io.opentracing.Tracer");
+      TracingConcurrencyStrategy.register(io.opentracing.util.GlobalTracer.get());
+    } catch (ClassNotFoundException | LinkageError e) {
+      LOGGER.debug("OpenTracing is not on classpath, skipping context propagation instrumentation");
```

Comment:
This should probably be an INFO. (I'm thinking WARN, but not sure.)

Reply:
Most of the cases tracing might not be present. I will change it to info then

Reply:
Yea, I didn't think of that. So, hmm, I'm thinking, in case of Thorntail: to make this usable, the `io.smallrye.faulttolerance` module (as in JBoss Modules) would have to have a dependency on the `io.opentracing.tracer` module. But if OpenTracing isn't configured by some fraction, then it wouldn't do anything useful anyway. So we'd have to make the module dependency optional. Then, the `io.opentracing.tracer` module would only be present if the user depends on one of the following fractions: `opentracing`, `opentracing-tracerresolver` (but that's internal), `jaeger`, or `microprofile-opentracing`. Then it would work as intended...

In the end, it seems DEBUG was the right choice. Sorry about the fuss :-)

Reply:
This is pretty much implementation detail which should work without user interference. I will revert back to debug.

I am not fully following proposed thorntail impl. Can you make it work for thorntail?

Reply:
Oh I can make it work with Thorntail no problem. I just had to go through this in my mind to understand how it's supposed to be used. Thanks for understanding :-)